### PR TITLE
Format

### PR DESCRIPTION
**Pull request by gaoxinge**
**What this PR does**:

- add blank between functions

**Which issue(s) this PR fixes**:

- None

**Special notes for your reviewer**:

- None

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: config/protocol_config.go [hunk 1]
```diff
@@ -38,11 +38,11 @@ func (c *ProtocolConfig) Prefix() string {
 }
 
 func loadProtocol(protocolsIds string, protocols map[string]*ProtocolConfig) []*ProtocolConfig {
-	returnProtocols := []*ProtocolConfig{}
+	var returnProtocols []*ProtocolConfig
```
Comment (zouyx) [inline config/protocol_config.go]:
```suggestion
	returnProtocols :=make([]*ProtocolConfig,0,len(protocols))
```
Specify the capacity and length is better

File: config/reference_config.go [hunk 2]
```diff
@@ -131,10 +129,11 @@ func (c *ReferenceConfig) Refer(_ interface{}) {
 			regUrl.SubURL = cfgURL
 		}
 	}
+
 	if len(c.urls) == 1 {
 		c.invoker = extension.GetProtocol(c.urls[0].Protocol).Refer(*c.urls[0])
 	} else {
-		invokers := []protocol.Invoker{}
+		var invokers []protocol.Invoker
```
Comment (zouyx) [inline config/reference_config.go]: As above

**Comment by AlexStocks** [comment]
@zouyx  u own this pr.

File: config/protocol_config.go (hunk 1 above)
Comment (gaoxinge) [reply config/protocol_config.go]: Nice advice. I specify the capacity when initializing slice.

File: config/reference_config.go (hunk 2 above)
Comment (gaoxinge) [reply config/reference_config.go]: As above.

File: config/reference_config.go [hunk 3]
```diff
@@ -121,10 +121,10 @@ func (c *ReferenceConfig) Refer(_ interface{}) {
 			}
 		}
 	} else {
-		//2. assemble SubURL from register center's configuration模式
+		// 2. assemble SubURL from register center's configuration模式
```
Comment (zouyx) [inline config/reference_config.go]: help change to english comment ,thank you.

**Comment by AlexStocks** [comment]
LGTM

**Pull request merged**